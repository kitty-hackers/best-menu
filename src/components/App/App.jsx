'use strict';

import { Layout } from 'antd';
import * as React from 'react';

const { Header, Sider, Content, Footer } = Layout;

function App() {
  return (
    <>
      <Layout className="site">
        <Sider>Sider</Sider>

        <Layout>
          <Header>Header</Header>
          <Content>Content</Content>
          <Footer>Footer</Footer>
        </Layout>
      </Layout>
    </>
  );
}

export default App;
