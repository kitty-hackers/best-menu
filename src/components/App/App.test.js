'use strict';

import * as React from 'react';
import * as renderer from 'react-test-renderer';

import App from './App';

const mock = {
  test: true,
};

it(`should be truely`, () => {
  const { test } = mock;

  expect(test).toBe(true);
});

it(`should render App snapshot`, () => {
  const app = renderer
    .create(<App />)
    .toJSON();

  expect(app).toMatchSnapshot();
});
