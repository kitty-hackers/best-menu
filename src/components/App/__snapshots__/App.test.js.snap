// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App snapshot 1`] = `
<section
  className="ant-layout ant-layout-has-sider site"
>
  <aside
    className="ant-layout-sider ant-layout-sider-dark"
    style={
      Object {
        "flex": "0 0 200px",
        "maxWidth": "200px",
        "minWidth": "200px",
        "width": "200px",
      }
    }
  >
    <div
      className="ant-layout-sider-children"
    >
      Sider
    </div>
  </aside>
  <section
    className="ant-layout"
  >
    <header
      className="ant-layout-header"
    >
      Header
    </header>
    <main
      className="ant-layout-content"
    >
      Content
    </main>
    <footer
      className="ant-layout-footer"
    >
      Footer
    </footer>
  </section>
</section>
`;
